src/components/MonthYear/MonthYear.jsx

```diff
@@ -72,18 +72,18 @@ class MonthYear extends React.Component {
     }
 
     return (
-      <div className={!isValid ? 'input-error-date' : undefined}>
-        <label>
-          {this.props.label}
+      <fieldset className={!isValid ? 'input-error-date' : undefined}>
+        <legend className="vads-u-font-size--base vads-u-font-weight--normal">
+          {this.props.label || 'Date'}
           {this.props.required && (
             <span className="form-required-span">(*Required)</span>
           )}
-        </label>
+        </legend>
         {errorSpan}
         <div
           className={isValid ? undefined : 'usa-input-error form-error-date'}
         >
-          <div className="usa-date-of-birth">
+          <div className="usa-date-of-birth usa-datefields clearfix-text">
             <div className="form-datefield-month">
               <Select
```

Comment:
These select menus and input have an extra CSS rule that causes them to get wider when the error class is applied. Can we remove this extra error state width rule so the selects and input stay the same width?

```css
.user-input-error select {
  width: calc(100% + 1.5rem);
}
```

![Screen Shot 2021-02-01 at 9 43 20 PM](https://user-images.githubusercontent.com/934879/106549671-15b71b80-64d7-11eb-90ac-06f447c4d278.png)

Reply:
CSS changes will need to happen in the VFST repo. My plan was to eventually get rid of the float and use flex-box to keep these inputs inline. I didn't think that change would be in scope with this work.

Reply:
Also, would it be better to spell out the full month name? I thought I remembered the screenreader trying to read the abbreviations and it sounded weird.

Reply:
If we can spell the names out, yes that would be great! Screen readers do a so so job of unpacking the abbreviations, and inconsistently at that.

Reply:
I can go with the CSS adjustment being a separate ticket (out of scope). I'll write a separate ticket for the CSS change work now, and reference it here.

Reply:
New issue for the CSS here: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/394

Reply:
Actually, I don't know if replacing the months with a full name would break stuff... I think it should be a separate PR.